docker-for-mac/osxfs.md

```diff
@@ -1,369 +0,0 @@
-are still evaluating which privileges to drop in the file system process to
-balance security and ease-of-use. `osxfs` performs no additional permissions
-checks and enforces no extra access control on accesses made through it. All
-processes in containers can access the same objects in the same way as the
-Docker user who started the containers.
-
-### Namespaces
-
-Much of the macOS file system that is accessible to the user is also available to
-containers using the `-v` bind mount syntax. The following command runs a container
-from an image called `r-base` and shares the macOS user's `~/Desktop/` directory as
-`/Desktop` in the container.
-
-```bash
-$ docker run -it -v ~/Desktop:/Desktop r-base bash
-```
-
-The user's `~/Desktop/` directory is now visible in the container as a directory
-under `/`.
-
-```
-root@2h30fa0c600e:/# ls
-Desktop	boot	etc	lib	lib64	media	opt	root	sbin	sys	usr
-bin	dev	home	lib32	libx32	mnt	proc	run	srv	tmp	var
-```
-
-By default, you can share files in `/Users/`, `/Volumes/`, `/private/`, and
-`/tmp` directly. To add or remove directory trees that are exported to Docker,
-use the **File sharing** tab in Docker preferences
-![whale menu](images/whale-x.png){: .inline} -> **Preferences** ->
-**File sharing**. (See [Preferences](index.md#file-sharing).)
-
-All other paths
-used in `-v` bind mounts are sourced from the Moby Linux VM running the Docker
-containers, so arguments such as `-v /var/run/docker.sock:/var/run/docker.sock`
-should work as expected. If a macOS path is not shared and does not exist in the
-VM, an attempt to bind mount it fails rather than create it in the VM. Paths
-that already exist in the VM and contain files are reserved by Docker and cannot
-be exported from macOS.
-
-> See **[Performance tuning for volume mounts (shared filesystems)](osxfs-caching.md)**
-> to learn about new configuration options available with the Docker 17.04 CE Edge release.
-
-### Ownership
-
-Initially, any containerized process that requests ownership metadata of an
-object is told that its `uid` and `gid` own the object. When any containerized
-process changes the ownership of a shared file system object, such as by using
-the `chown` command, the new ownership information is persisted in the
-`com.docker.owner` extended attribute of the object. Subsequent requests for
-ownership metadata return the previously set values. Ownership-based permissions
-are only enforced at the macOS file system level with all accessing processes
-behaving as the user running Docker. If the user does not have permission to
-read extended attributes on an object (such as when that object's permissions
-are `0000`), `osxfs` attempts to add an access control list (ACL) entry that
-allows the user to read and write extended attributes. If this attempt fails,
-the object appears to be owned by the process accessing it until the extended
-attribute is readable again.
-
-### File system events
```

Comment:
These parts also still seem relevant

Reply:
All of this but the name of the extended attribute for FUSE is still true. Why was it removed? Just ran into this last week on latest edge.